backwards-compatability/node10/tsconfig.json

```diff
@@ -9,10 +9,10 @@
   ],
   "references": [
     {
-      "path": "../../packages/opentelemetry-sdk-node"
+      "path": "../../packages/opentelemetry-sdk-node-all"
```

Comment:
Just a general (non-blocking) comment/observation.

As we are renaming folders to also rename the packages do we "really" need the leading ```opentelemetry-``` prefix on every folder? As this only really contains packages for ```opentelemetry``` ? 😀 

Rationale: 
Really just making the folder depth shorter and helping to avoid excessive folder length (especially for developers using windows and the path limitation for embedded node_modules etc -- not that this alone resolves that it just helper (a little))

Reply:
I 💯 agree with this. I'm not sure where this convention comes from because it is before my time. Maybe @obecny knows? It might just be a holdover from when we were porting a lot of things from census

Reply:
Totally taken from opencensus (https://github.com/census-instrumentation/opencensus-node/tree/master/packages), i would not be surprised if it was a internal conventions from google and we just re-used it for otel

Reply:
it was like this before me too, probably no1 is here anymore who could answer this question

Reply:
I don't think we want to tackle this right now no ? Its only internal naming

Reply:
probably in another PR

Reply:
IMHO: As we are already changing the folders in this PR then it makes sense to do this once rather than take on the tech debt to do it later (which probably wont happen) -- signing off but my preference is to do this as part of the existing change.

Reply:
I'd also want to see "opentelemetry-" prefix disappear, but it's probably fine to do it in another PR.

Reply:
I created an issue for this (https://github.com/open-telemetry/opentelemetry-js/issues/2376), i will tackle this when this PR get merged